### PR TITLE
Add Journey into a Beatmap World Art Contest news post

### PR DESCRIPTION
**Pull request by Ephemeralis**
It's like isekai, but for beatmaps. And not terrible.

Note: the contest listing is not currently visible, but is prepared. Need to toggle it before merging after approval.

### CODE REVIEW

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: news/2023/2023-03-23-mascot-in-a-beatmap-art-contest.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+- **Entries MUST meet the resolution requirements for a main-menu piece — namely 2732x1536.** The contest system won't let you mess this up any more! Progress!
+- **Artists should use the custom templates provided ([PSD template](https://assets.ppy.sh/events/fanart/templates/osu%21%20main%20menu%202732x1536.psd?2017) / [PNG template](https://assets.ppy.sh/events/fanart/templates/osu%21%20main%20menu%202732x1536.png?2017)) when composing their piece as a guideline to understand where the osu! cookie will be placed on the main menu and during the voting process.** Very simply: anything in the big white circle won't be seen during the voting. Don't leave a big white hole in your piece either, please!
+- **Do not leave the dotted osu! cookie template visible in your submission.** The osu! cookie template does not feature in any known beatmaps and it should also not feature in your works. Don't be a bad apple.
```
[the remark this answers is not in the document]
Comment (Ephemeralis) [reply news/2023/2023-03-23-mascot-in-a-beatmap-art-contest.md:49]: don't deny me this catharsis

File: news/2023/2023-03-23-mascot-in-a-beatmap-art-contest.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+layout: post
+title: "Mascot in a Beatmap Art Contest"
+date: 2023-03-23 16:00:00 +0000
+---
+
+Swap out your pen nibs and prepare your eyeballs — it's time for 2023's first art contest!
+
+![](https://assets.ppy.sh/contests/175/header.jpg)
+
+Our much-beloved art contests are back again for 2023, and we're starting things off with one of the themes of all time. So much of a theme in fact, that it is the least-creatively named but most creative art contest we've ever held: the *Mascot in a Beatmap Art Contest*.
```
[the remark this answers is not in the document]
Comment (Walavouchey) [reply news/2023/2023-03-23-mascot-in-a-beatmap-art-contest.md]: snakes on a plane reference? really? that sounds far more removed from the contest's concept and from anything osu! in general than what i could've thought initially.

[the remark this answers is not in the document]
Comment (peppy) [reply news/2023/2023-03-23-mascot-in-a-beatmap-art-contest.md]: I'm not set on any particular name, so will leave it up to @Ephemeralis

File: news/2023/2023-03-23-mascot-in-a-beatmap-art-contest.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+layout: post
+title: "Mascot in a Beatmap Art Contest"
+date: 2023-03-23 16:00:00 +0000
+---
+
+Swap out your pen nibs and prepare your eyeballs — it's time for 2023's first art contest!
+
+![](https://assets.ppy.sh/contests/175/header.jpg)
+
+Our much-beloved art contests are back again for 2023, and we're starting things off with one of the themes of all time. So much of a theme in fact, that it is the least creatively named but most creative art contest we've ever held: the *Mascot in a Beatmap Art Contest*.
```
Comment (peppy) [inline news/2023/2023-03-23-mascot-in-a-beatmap-art-contest.md:11]: > starting things off with one of the themes of all time

is this correct english? feels a bit weird.

Comment (Ephemeralis) [reply news/2023/2023-03-23-mascot-in-a-beatmap-art-contest.md:11]: it is a meme

File: news/2023/2023-03-23-mascot-in-a-beatmap-art-contest.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Ephemeralis) [reply news/2023/2023-03-23-mascot-in-a-beatmap-art-contest.md]: changed it to Journey into a Beatmap World after some discussion

**Review by Walavouchey — APPROVED** [review]

File: news/2023/2023-03-23-mascot-in-a-beatmap-art-contest.md (hunk 3 above)
Comment (peppy) [reply news/2023/2023-03-23-mascot-in-a-beatmap-art-contest.md]: okay i'll trust your meme knowledge today.

Comment (Walavouchey) [reply news/2023/2023-03-23-mascot-in-a-beatmap-art-contest.md]: it works well enough https://github.com/ppy/osu-wiki/pull/9012#discussion_r1146451748

**Pull request merged**